tailwind.config.cjs

```diff
@@ -7,5 +7,7 @@ const tailwindcssRadix = require('tailwindcss-radix')
 module.exports = {
   theme: themeConf,
   content: ['./packages/**/*.{js,ts,jsx,tsx}', './documentation/**/*.{js,ts,jsx,tsx,mdx}'],
-  plugins: [sparkPlugins.animations(), sparkPlugins.sizings(), tailwindcssRadix()],
+  plugins: [sparkPlugins.animations(), sparkPlugins.sizings(), tailwindcssRadix({
+    variantPrefix: 'spark',
```

Comment:
Beware that this will impact every components that already this plugin classes

Reply:
Yes. I think that is not being used for now in the `main` branch. What do you think of using `spark` as prefix instead of `radix` ?

Reply:
sounds good to me 👍